eisbuk-admin/cypress/integration/create_slot_spec.ts

```diff
@@ -92,4 +87,35 @@ describe("Create slot", () => {
     cy.get("[aria-label='Additional slot notes']").type("some notes");
     cy.get("[aria-label='Confirm slot creation']").click();
   });
+
+  it("submits an invalid time format", () => {
```

Comment:
I would write this as "showa 'invalid time format' invalidation error" (doesn't technically submit)